doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,74 @@
+# Script-Languages-Developer-Sandbox User Guide
+
+## Overview
+
+This packages aims to create a virtual machine image, in different formats, which can be used to build easily Exasol's script-languages-container, which are the runtime container for UDF's.
+The idea is to set-up an AWS EC-2 instance, using cloudformation templates, then install all dependencies via _Ansible_, generate an AMI image based on the final EC-2 instance, and finally export the AMI image to the virtual image formats.
+The virtual machine image provides:
+* The script-languages-release repositories
+* All necessary dependencies to execute _exaslct_ in script-languages-release (This includes a correctly configured docker runtime)
+* A running Jupyterlab instance which is automatically started during boot of the vm
+
+## Objective
+
+The objective is to have a complete VM ready, including all dependencies, which can be used as base to customize the existing template script-languages-container (e.g. python-3.x-minimal templates, the r-minimal templates); or, if needed, to create new script-languages-container from scratch.
+The VM is per default large enough to build and test script-languages-container.
+
+## Provided medias
+
+### AMI
+
+The AMI id is linked in the release notes and can be used to start an EC2-instance in your AWS account.
+Following name format is used for the AMI: "_Exasol-SLC-Developer-Sandbox-**version**_", e.g. "_Exasol-SLC-Developer-Sandbox-5.0.0_"
+
+### VM image
+
+Currently two VM formats are supported:
+
+| Format      | Description                       |
+| ----------- | --------------------------------- |
+| VMDK        | VMware Virtual Machine Disk       |
+| VHD         | Virtual Hard Disk by Microsoft    |
+
+The links to the images are stored in the release notes.  
+For security reasons the formats are currently stored in a private AWS S3 Bucket, and not publicly available.
```

Comment:
Hm, didn't we want to make the released files public

Reply:
as discussed offline, will make the bucket public. removed this comment.